cmd/distrogen/templates/Makefile.go.tmpl

```diff
@@ -109,14 +109,15 @@ collector-build: clean-collector $(COLLECTOR_BINARY_NAME)
 
 .PHONY: ocb-generate
 ocb-generate: $(OCB_BIN)
-	$(SET_GO_BIN_PATH) $(if $(USE_GO_PROXY),GOPROXY=$(USE_GO_PROXY),) $(OCB_BIN) --skip-compilation --verbose --config manifest.yaml
+	$(SET_GO_BIN_PATH) $(if $(USE_GO_PROXY),GOPROXY=$(USE_GO_PROXY) GOSUMDB=off,) $(OCB_BIN) --skip-compilation --verbose --config manifest.yaml
 
 # Using OCB and the local manifest file, build a collector.
 $(COLLECTOR_BINARY_NAME): ocb-generate
 	cd $(COLLECTOR_BUILD_TEMP_DIR) && \
 		GOWORK=off \
 		CGO_ENABLED=$(COLLECTOR_CGO) \
 		GOOS=$(TARGETOS) GOARCH=$(TARGETARCH) \
+		$(if $(USE_GO_PROXY),GOPROXY=$(USE_GO_PROXY) GOSUMDB=off,) \
```

Comment:
I think the Go installs happen in the `ocb-generate` target, so I don't think we need to re-export `GOPROXY` here on line 120. Can we try removing this line and see if it still works?